aw_watcher_window/main.py

```diff
@@ -42,7 +42,7 @@ def main():
     logger.info("aw-watcher-window started")
     sleep(1)  # wait for server to start
     with client:
-        heartbeat_loop(client, bucket_id, poll_time=args.poll_time, exclude_title=args.exclude_title)
+        heartbeat_loop(client, bucket_id, poll_time=args.poll_time, exclude_title=args.exclude_title or config.getboolean("exclude_title"))
```

Comment:
This is incorrect, an cli argument should always take precedence over a config setting.
You should handle it in the same way as the poll-time argument, pass the config value to parse_args and only use it as the default.